### PR TITLE
[OSCD] 109: Pester.bat 

### PR DESCRIPTION
**Pull request by Vasilisa-L**
Detects code execution via Pester.bat (Pester - Powershell Modulte for testing) 
#1014

### CODE REVIEW

File: rules/windows/process_creation/win_susp_pester.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        CommandLine|contains|all:
+            - 'Pester'
+            - 'Get-Help'
+    cmd_execution:
+        Image|endswith: '\cmd.exe'
+        CommandLine|contains|all: 
+            - 'pester'
+            - ';'
+    get_help:
+        CommandLine|contains:
```
Comment (aw350m33d) [inline rules/windows/process_creation/win_susp_pester.yml:27]: The [LOLBIN page also says](https://github.com/LOLBAS-Project/LOLBAS/blob/94368c1e69a6ce5ce812f2b331c99b89a63791b9/yml/OSScripts/pester.yml#L7) that you can use just a question mark. And in the [old repository](https://github.com/api0cradle/LOLBAS/blob/master/OSScripts/pester.md), it is additionally said that there is still a help key without a slash

It looks like this:
image

Can you additionally check these parameters?

**Comment by aw350m33d** [comment]
Wow! 😮 
I am impressed by the quality of your approach! 🥇 

You made a great rule! It can be merged as soon as we find out if there are additional command line parameters.

File: rules/windows/process_creation/win_susp_pester.yml (hunk 1 above)
Comment (Vasilisa-L) [reply rules/windows/process_creation/win_susp_pester.yml:27]: Yeah, my fault
Just 'help' and '?' work as well
Fixed in the rule

Comment (aw350m33d) [reply rules/windows/process_creation/win_susp_pester.yml:27]: Cool! Thank you! 👍🏻

**Review by aw350m33d — CHANGES_REQUESTED** [review]
You also detect the launch directly through the PowerShell. Therefore, I suggest adding an additional technique `T1059.001` from the ʻExecution` tactic.

**Comment by Vasilisa-L** [comment]
> You also detect the launch directly through the PowerShell. 

@aw350m33d 
Actually, i didn't test launch directly via Powershell. Powershell Pester.psm1 is triggered by pester.bat 
ComandLine looks like this `PowerShell -NonInteractive -NoProfile -ExecutionPolicy Bypass -Command "& Import-Module 'c:\Program Files\WindowsPowerShell\Modules\Pester\3.4.0\bin\..\Pester.psm1'; & { Get-Help "$null; notepad"}"`

**Comment by aw350m33d** [comment]
Yup! And this command line starts a PowerShell process to execute the code, doesn't it?

**Comment by aw350m33d** [comment]
Julia, great job! 

I do not know why the tests do not pass, but I like the rule and I believe that it can be merged. @yugoslavskiy

**Comment by yugoslavskiy** [comment]
Thank you very much, @Vasilisa-L! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**